.gitignore

```diff
@@ -13,3 +13,6 @@ npm-debug.log
 yarn-error.log
 /.idea
 /.vscode
+package-lock.json
+package.json
```

Comment:
sao e lại ignore 2 file này?
2 file này dùng để làm gì?

Reply:
package.json là file chứa thông giúp người dùng biết được để vận hành được project cần đến những modules nào.
sử dụng package-lock.json thay vì package.json để cài đặt các modules do package-lock.json đã chỉ định rõ phiên bản, location, mã băm toàn vẹn integrity cho mỗi module và từng dependencies của nó